### PR TITLE
[AC-8824]: Update location_regional field with custom migration

### PR DESCRIPTION
**Pull request by kenware**
[AC-8824](https://masschallenge.atlassian.net/browse/AC-8824)

How to test:
- Confirm that the migration runs on your local machine without error
- If necessary, you can run an SQL on accelerator_startup on a field `location_regional`; to ensure that Items changed are persisted
-

### CODE REVIEW

File: accelerator/migrations/0041_correct_data_in_accelerator_startup.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.16 on 2021-05-11 15:00
+
+from django.db import migrations
+
+
+MASSACHUSETTS = 'Massachusetts'
+CALIFONIA = 'California'
```
Comment (jonkiparsky) [inline accelerator/migrations/0041_correct_data_in_accelerator_startup.py:7]: Typo, s/b `CALIFORNIA`

File: accelerator/migrations/0041_correct_data_in_accelerator_startup.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+NEW_YORK = 'New York'
+RHODE_ISLAND = 'Rhode Island'
+CALIFONIA_DATA_LIST = [
```
Comment (jonkiparsky) [inline accelerator/migrations/0041_correct_data_in_accelerator_startup.py:10]: Shouldn't this be `MASSACHUSETTS_DATA_LIST`?

**Review by jonkiparsky — CHANGES_REQUESTED** [review]
Two trivial issues

File: accelerator/migrations/0041_correct_data_in_accelerator_startup.py (hunk 2 above)
Comment (kenware) [reply accelerator/migrations/0041_correct_data_in_accelerator_startup.py]: Thanks for spotting these @jonkiparsky

**Review by jonkiparsky — APPROVED** [review]
Reviewing the data on prod and the [dataset used by the CRS library](https://github.com/country-regions/country-region-data/blob/master/data.json) I see that there are many regional names in the DB which are not normalized correctly. However, having consulted with Jack the current changes are sufficient for him, so approving.

**Review by iankigen — APPROVED** [review]
Code/QA looks good
- Disparate values no longer exist for the same intended value in this field. ✅ 
- should be done via a data migration. ✅

**Pull request merged**